js-intro.js

```diff
@@ -55,7 +106,19 @@ var notAString2 = 42
 
 var toonimals = [ {name: "Itchy", animal: "mouse"}, {name: "Stimpy", animal: "cat"}, {name: "Daffy", animal: "duck"}, {name: "Scratchy", animal: "cat"}, {name: "Ren", animal: "dog"}, {name: "Felix", animal: "cat"}]
 
-
+var onlyCats = toonimals.filter(value => value.animal === "cat")
+console.log(onlyCats)
 
 // --------------------7) Using the toonimals variable from #6, create a function that returns only the names of the non-cats.
 // Expected output: "Itchy" "Daffy" "Ren"
+
+var toonimals = [ {name: "Itchy", animal: "mouse"}, {name: "Stimpy", animal: "cat"}, {name: "Daffy", animal: "duck"}, {name: "Scratchy", animal: "cat"}, {name: "Ren", animal: "dog"}, {name: "Felix", animal: "cat"}]
+
+var onlyNonCatNames = []
+toonimals.filter((value) => {
+if(value.animal!=="cat"){
+onlyNonCatNames.push(value.name)
+}
+})
```

Comment:
Same here, the log looks good but we need it to be wrapped in a function instead of a variable